src/components/Trade/hooks/useSwapper/useSwapper.test.tsx

```diff
@@ -19,7 +19,11 @@ jest.mock('context/PluginProvider/PluginProvider')
 function setup({
   action = TradeActions.SELL,
   approvalNeededBoolean = false,
-  quote = { rate: '1.2' },
+  quote = {
+    rate: '1.2',
+    buyAsset: USDC,
+    sellAsset: WETH,
```

Comment:
smelly smelly - without understanding the context more - is this the boundary layer of the swapper? 

we should be using something similar to a caip adapter for the swappers - i.e. pass around caips as much as possible internally, and at the boundary layer map from caips to swapper specific symbols (🤮)

please make another ticket to address this

Reply:
deerrrrrrp my bad these are asset consts in tests, disregard me i suck cocks